src/app/public/modules/href/fixtures/href-resolver-fixture.service.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@angular/core';
+import { SkyHrefResolver } from '../href-resolver';
+import { AppRoute } from '../types/app-route';
+
+@Injectable()
+export class HrefResolverFixtureService implements SkyHrefResolver {
+  public resolveHref(url: string): Promise<AppRoute> {
+    if (url.startsWith('test://')) {
```

Comment:
So that future developers may make changes to this feature and break these tests, this test URL resolver should be simplified down to something like a simple string match. That way the developer isn't also trying to troubleshoot a regular expression.

```
switch (url) {
  case 'test://foo':
    return 'https://example.com/bar';
}
```